src/client/src/components/Quiz/ProgressBar/ProgressBar.css

```diff
@@ -0,0 +1,13 @@
+.progress-bar-container{
+    width: 100%;
+    padding: 2%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    justify-self: flex-start;
+}
+
+.MuiLinearProgress-root{
+    width: 90%;
```

Comment:
My mentor once told me that it's better to use fixed sizes than % in order not to waste CPU

Reply:
but then it wouldn't be responsive, no?